### PR TITLE
2.3.0

### PR DESCRIPTION
**Pull request by HeavyTony2**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes: #26 
Closes: #25

### CODE REVIEW

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

**Comment by HeavyTony2** [comment]
ping @ap--

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "2.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b7500c145135836a952e3de3a80881d9654e327a29c852c9265607f5c449235c
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/flask_caching-{{ version }}.tar.gz
```
Comment (ap--) [inline recipe/meta.yaml]: Please make this small modification:

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_').lower() }}-{{ version }}.tar.gz
```

**Review by ap-- — CHANGES_REQUESTED** [review]
Thank you for your contribution @HeavyTony2 and @rxm7706 

Please commit the small suggestion and then this is good to be merged.

**Pull request merged**